bundles/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/FreeboxApiManager.java

```diff
@@ -493,11 +492,8 @@ private static String hmacSha1(String key, String value) throws FreeboxException
             // Compute the hmac on input data bytes
             byte[] rawHmac = mac.doFinal(value.getBytes());
 
-            // Convert raw bytes to Hex
-            byte[] hexBytes = new Hex().encode(rawHmac);
-
-            // Covert array of Hex bytes to a String
-            return new String(hexBytes, StandardCharsets.UTF_8);
+            // Convert raw bytes to a String
+            return HexUtils.bytesToHex(rawHmac);
```

Comment:
I have the feeling this will not be the same result.
The new call will add a space between each hexa value while the previous code did not.

Reply:
It will work - check it out. I wrote a unit test for each change.

```
    @Test
    public void testFreebox() {
        byte[] rawSubject = "test".getBytes(StandardCharsets.UTF_8);

        byte[] hexBytes = new Hex().encode(rawSubject);
        String subject = new String(hexBytes, StandardCharsets.UTF_8);

        assertThat(subject, is(HexUtils.bytesToHex(rawSubject)));
    }
```

Reply:
Ok, if you are sure, that is fine for me.

Reply:
@lolodomo Then please remove your "changes requested" review :-)

Reply:
I don't know how to suppress it !
I approved it even if I check only the part relative to the Freebox binding.

Reply:
Seems you made it :-)

Reply:
Yes, I approved it globally but only reviewed one part.